### PR TITLE
Bugfix: make deepstack detector thread-safe

### PR DESCRIPTION
**Pull request by skrashevich**

### CODE REVIEW

**Comment by skrashevich** [comment]
[35d23a0](https://github.com/blakeblackshear/frigate/pull/7055/commits/35d23a004419c8a3ee5b14a255a9b07d383a07eb) improve detection performance (Reduces CPU utilisation) and improve inference speed

File: requirements-wheels.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@ click == 8.1.*
 Flask == 2.3.*
 faster-fifo == 1.4.*
 imutils == 0.5.*
+httpx == 0.24.*
```
Comment (blakeblackshear) [inline requirements-wheels.txt:5]: Why introduce a new dependency here? Seems like requests is fine for this use case.

Comment (skrashevich) [reply requirements-wheels.txt:5]: python `requests` isn't really thread-safe :(

Comment (blakeblackshear) [reply requirements-wheels.txt:5]: Where are you seeing that? Thread-safe is specifically mentioned in the docs: https://docs.python-requests.org/en/v2.9.1/

What errors are you/others seeing that necessitate this change?

**Pull request closed without merging**